lib/commentTreeUtils.es6.js

```diff
@@ -6,7 +6,7 @@ import { COMMENT_LOAD_MORE } from '../models2/thingTypes';
 export function treeifyComments(comments=[]) {
   const commentDict = {};
   comments.forEach(c => {
-    commentDict[c.uuid] = c;
```

Comment:
I think treeify used to work on the models directly and got changed to json without proper testing. It will be a great day when we have tests in here